server/Knowledge_checklist.sql

```diff
@@ -0,0 +1,100 @@
+drop table if exists students cascade;
+drop table if exists mentors cascade;
+drop table if exists region;
+drop table if exists learning_objectives cascade;
+drop table if exists mapping_skills cascade;
+drop table if exists tech_skills cascade;
+drop table if exists competency_level cascade;
+
+CREATE TABLE students (
```

Comment:
The `students` and `mentors` tables contain exactly the same columns, my suggestion is to merge the two together into a `users` table. Try to think of ideas on how to make a difference between students and mentors if they are both stored in the same table.

Reply:
EM added new line 'subjects' on Mentors table